generators/python_functions/url_keyword_parser/__init__.py

```diff
@@ -0,0 +1,96 @@
+import re
+from urllib.parse import urlparse
+from nltk.corpus import words
+from nltk.corpus import stopwords
+from typing import List
+from pydantic import BaseModel
+
+INPUT_EXAMPLE = {
+    "url": "https://www.tagesschau.de/ausland/streiks-frankreich-137.html",
+    "includeDomain": True,
+    "includeParameter": True,
+    "checkValidUrl": True,
+    "removeNoneEnglish": False,
+    "removeStopwords": True,
+    "removeHexLike": True,
+    "textSeperator": ", ",
+    "splitRegex": "\W",
+    "wordWhiteList": ["heuristics"]
```

Comment:
differnt default parameter maybe
1. Maybe an English link since german isn't fully supported by us
2. heuristics was just an example I used in my testing

```suggestion
    "url": "https://stackoverflow.com/questions/19560498/faster-way-to-remove-stop-words-in-python",
    "includeDomain": True,
    "includeParameter": True,
    "checkValidUrl": True,
    "removeNoneEnglish": False,
    "removeStopwords": True,
    "removeHexLike": True,
    "textSeperator": ", ",
    "splitRegex": "\W",
    "wordWhiteList": None
```